models/HubSpotAuthError.ts

```diff
@@ -23,3 +23,7 @@ export class HubSpotAuthError extends Error {
       undefined;
   }
 }
+
+export function isHubSpotAuthError(err: unknown): err is HubSpotAuthError {
```

Comment:
Do we need this util? It looks like we're only using it in one place. If we do, I think it might be better off living in another file rather than in the models/ folder

Reply:
We can ditch it, I was using it in more places initially.